### ISSUE TITLE
Create tiny image instead of text when creating a missing blob

### ISSUE THREAD

**Issue by mauritsvanrees** [collective/experimental.gracefulblobmissing#5]
When using this package in development, I get lots of errors in the log like this:

```
2021-10-20 11:06:17,168 ERROR   [root:132][waitress-1] could not scale "<plone.namedfile.file.NamedBlobImage object at 0x12775df20 oid 0x26756c in <Connection at 108173610>>" of 'http://localhost:8080/es/front-page'
Traceback (most recent call last):
  File "/Users/maurits/shared-eggs/cp38/plone.app.tiles-3.2.0-py3.8.egg/plone/app/tiles/imagescaling.py", line 123, in create
    result = scaleImage(
  File "/Users/maurits/shared-eggs/cp38/plone.scale-3.1.2-py3.8.egg/plone/scale/scale.py", line 57, in scaleImage
    image = PIL.Image.open(image)
  File "/Users/maurits/shared-eggs/cp38/Pillow-6.2.2-py3.8-macosx-10.15-x86_64.egg/PIL/Image.py", line 2818, in open
    raise IOError("cannot identify image file %r" % (filename if filename else fp))
OSError: cannot identify image file <_io.BytesIO object at 0x127768f40>
```

The site works fine, I see no errors in the browser, but the errors in the log/foreground are annoying.
Currently the package [sets a string in a missing blob](https://github.com/collective/experimental.gracefulblobmissing/blob/009f1db9e65b35cb5d222b7bcf572c094d017b68/experimental/gracefulblobmissing/patches.py#L244): "File created by experimental.gracefulblobmissing."

Would it be okay to fill the 'fake' blob with a small image instead? I guess just the Plone logo, either in PNG or JPEG. Or maybe just one pixel.

**Comment by ale-rt** [comment]
I did something similar in the past: https://github.com/collective/experimental.gracefulblobmissing/tree/auto-create-images

**Comment by thet** [comment]
I had a need for this feature and did a rebase on current master on the branch auto-create-images--rebased.
PR here: https://github.com/collective/experimental.gracefulblobmissing/pull/6

**Comment by mauritsvanrees** [comment]
Released a few days ago in version 1.0.
